modules/video/src/lkpyramid.cpp

```diff
@@ -419,11 +425,11 @@ void cv::detail::LKTrackerInvoker::operator()(const Range& range) const
             }
         }
 
-#if CV_SSE2
+#if CV_SSE2 || CV_VSX
         float CV_DECL_ALIGNED(16) A11buf[4], A12buf[4], A22buf[4];
-        _mm_store_ps(A11buf, qA11);
-        _mm_store_ps(A12buf, qA12);
-        _mm_store_ps(A22buf, qA22);
+        v_store_aligned(A11buf, qA11);
+        v_store_aligned(A12buf, qA12);
+        v_store_aligned(A22buf, qA22);
         iA11 += A11buf[0] + A11buf[1] + A11buf[2] + A11buf[3];
```

Comment:
Probably we should use [`v_reduce_sum()`](https://github.com/opencv/opencv/blob/4.1.1/modules/core/include/opencv2/core/hal/intrin_cpp.hpp#L1023-L1036) here instead of temporary buffer.